tutorials/polkadot-sdk/build-a-blockchain/simulate-a-network.md

```diff
@@ -0,0 +1,243 @@
+
+To demonstrate this concept, you'll simulate a network environment using two nodes running on a single computer, each configured with different accounts and keys. Throughout this tutorial, you'll gain practical insight into the functionality of the authority consensus model by observing how these two predefined accounts, serving as authorities, enable the nodes to produce blocks.
+
+By completing this tutorial, you will accomplish the following objectives:
+
+- Start a blockchain node using a predefined account
+- Learn the key command-line options used to start a node
+- Determine if a node is running and producing blocks
+- Connect a second node to a running network
+- Verify peer computers produce and finalize blocks
+
+## Prerequisites
+
+- Installed and configured Rust on your system. Refer to the [Installation]() guide for detailed instructions on installing Rust and setting up your development environment
+- Completed the [Build a Local Blockchain](#build-a-local-blockchain) guide and have the [Polkadot-SDK Solochain Template](https://github.com/paritytech/polkadot-sdk-solochain-template){target=\_blank} installed on your local machine
+
+## Start the First Blockchain Node
+
+This tutorial demonstrates the fundamentals of a private network using a predefined chain specification called local and two preconfigured user accounts. You'll simulate a private network by running two nodes on a single local computer, using accounts named `Alice` and `Bob`.
+
+Follow these steps to start your first blockchain node:
+   
+1. Navigate to the root directory where you compiled the Polkadot-SDK Solochain Template
+   
+2. Clear any existing chain data by executing:
+    ```bash
+    ./target/release/solochain-template-node purge-chain --base-path /tmp/alice --chain local
+    ```
+
+    When prompted to confirm, type `y` and press *Enter*. This step ensures a clean start for your new network.
+
+3. Launch the first blockchain node using the `Alice` account:
+    ```bash
+    ./target/release/solochain-template-node \
+    --base-path /tmp/alice \
+    --chain local \
+    --alice \
+    --port 30333 \
+    --rpc-port 9945 \
+    --node-key 0000000000000000000000000000000000000000000000000000000000000001 \
+    --validator
+    ```
+
+### Review the Command-Line Options
+
+Before proceeding, examine the key command-line options used to start the node:
+
+- `--base-path` - specifies the directory for storing all chain-related data
+- `--chain` - defines the chain specification to use
+- `--alice` - adds the predefined keys for the `Alice` account to the node's keystore. This account is used for block production and finalization
+- `--port` - sets the listening port for peer-to-peer (p2p) traffic. Different ports are necessary when running multiple nodes on the same machine
+- `--rpc-port` - specifies the port for incoming JSON-RPC traffic via WebSocket and HTTP
+- `--node-key` - defines the Ed25519 secret key for libp2p networking
```

Comment:
Just double checking, I think nodes no longer auto produce keys anymore, hence this command? There is also an unsafe flag that can be used to auto-gen a node key for this purpose as well (`--unsafe-force-node-key-generation`)

Reply:
Actually, in order to have a consistent p2p address, your approach I think makes more sense.

Reply:
I struggled a bit with that, but I ended up hardcoding the key to maintain the p2p address of at least the first node that works as a bootnode for the other ones